app/src/app/components/routes/Router.tsx

```diff
@@ -22,6 +22,9 @@ export const Router = () => {
         </Route>
         {/* <Route path="accessibility" element={<Accessibility />} /> */}
         <Route path="redirect" element={<Redirect />} />
+        <Route element={<ProtectedRoute />}>
```

Comment:
confused why a login route would be protected

Reply:
Login is a confusing term here.. maybe rename to github equivalent `DeviceActivation.tsx`